### ISSUE TITLE
Firebase Auth: Signature wasn't verified successfully

### ISSUE THREAD

**Issue by BartuCulha** [PrimeDAO/prime-deals-dapp#808]
Steps to reproduce:
1.Navigate here: https://prime-deals-staging.vercel.app/deal/rVPs42adMgMMXNWqEUCgMo
2.Connect Wallet: 0xE834627cDE2dC8F55Fe4a26741D3e91527A8a498
3. Sign Message

After signing the message I have received these two errors:
### 1
ERROR [Prime Deals] Signature wasn't verified successfully vendors-319a6989.aebebf5e99f9a872fb4e.bundle.js:2:43093
    error https://prime-deals-staging.vercel.app/vendors-319a6989.aebebf5e99f9a872fb4e.bundle.js:2
    l https://prime-deals-staging.vercel.app/vendors-319a6989.aebebf5e99f9a872fb4e.bundle.js:2
    logMessage https://prime-deals-staging.vercel.app/app-ba75c428.5e0fb4011ae132f17d36.bundle.js:1
    handleFailure https://prime-deals-staging.vercel.app/app-ba75c428.5e0fb4011ae132f17d36.bundle.js:1
    c https://prime-deals-staging.vercel.app/app-ba75c428.5e0fb4011ae132f17d36.bundle.js:1
    i https://prime-deals-staging.vercel.app/vendors-319a6989.aebebf5e99f9a872fb4e.bundle.js:2
    publish https://prime-deals-staging.vercel.app/vendors-319a6989.aebebf5e99f9a872fb4e.bundle.js:2
    signInToFirebase https://prime-deals-staging.vercel.app/app-ba75c428.5e0fb4011ae132f17d36.bundle.js:1
### 2
ERROR [Prime Deals] Authentication failed vendors-319a6989.aebebf5e99f9a872fb4e.bundle.js:2:43093

**Comment by dkent600** [comment]
@BartuCulha Does this really have anything to do with that particular deal?  Are you able to reproduce this elsewhere?

**Comment by BartuCulha** [comment]
@dkent600 Yes Eugenio where showing me this deal on his computer, and the issue reoccured

https://prime-deals-staging.vercel.app/deal/8DQsYTMdA8JaXuAuwCgucT

**Comment by BartuCulha** [comment]
Able to reproduce on this deal: https://prime-deals-staging.vercel.app/deal/nKneWix9zYmP1yWBhJEqdP

Steps to reproduce:
-  Switch metamask accounts on this deal dashboard


Console log:
Failed to load resource: net::ERR_BLOCKED_BY_CLIENT
us-central1-prime-deals-staging.cloudfunctions.net/verifySignedMessageAndCreateCustomToken:1          Failed to load resource: the server responded with a status of 401 ()
vendors-319a6989.aebebf5e99f9a872fb4e.bundle.js:2 ERROR [Prime Deals] Signature wasn't verified successfully
e.error @ vendors-319a6989.aebebf5e99f9a872fb4e.bundle.js:2
vendors-319a6989.aebebf5e99f9a872fb4e.bundle.js:2 ERROR [Prime Deals] Authentication failed

**Comment by dkent600** [comment]
Please try again with new changes
